cost-analyzer/values.yaml

```
@@ -1864,14 +1864,17 @@ clusterController:
   imagePullPolicy: IfNotPresent
   priorityClassName: ""
   tolerations: []
-
   ## Annotations to be added for cluster controller template
   annotations: {}
   labels: {}
   securityContext: {}
   resources: {}
   affinity: {}
   nodeSelector: {}
+  primaryKubecostURL: ""  # URL for secondary clusters to connect to primary Kubecost
+  kubecostAPIKey: ""  # API Key for secondary clusters to authenticate with primary
```

```suggestion
  kubecostAPIKey: ""  # API Key for secondary clusters to authenticate with primary (ex: https://kubecost.myorganization.com)
```

Provide an example in the comments. Protocol should be included in the URL.

---

@mittal-ishaan Oops sorry. The comment should be for `primaryKubecostURL`, not `kubecostAPIKey`